app/templates/components/public/ticket-list.hbs

```diff
@@ -29,10 +29,28 @@
                 </div>
                 <div id="{{ticket.id}}_discount">
                   {{currency-symbol eventCurrency}} {{format-number (sub ticket.price ticket.discount)}}
+                  {{#if taxInfo}}
+                    <small class="ui gray-text small">
+                      + {{currency-symbol eventCurrency}} {{format-number (sub ticket.taxIncurred ticket.price)}}
+                    </small>
+                    <div>
+                      <small class="ui gray-text tiny aligned right">({{taxInfo.name}})</small>
+                    </div>
+                  {{/if}}
                 </div>
               </td>
             {{else}}
-              <td id="{{ticket.id}}_price">{{currency-symbol eventCurrency}} {{format-number ticket.price}}</td>
+              <td id="{{ticket.id}}_price">
+                {{currency-symbol eventCurrency}} {{format-number ticket.price}}
+                {{#if taxInfo}}
+                  <small class="ui gray-text small">
+                    + {{currency-symbol eventCurrency}} {{format-number (sub ticket.taxIncurred ticket.price)}}
```

Comment:
@uds5501 I think you should add a condition here to remove displaying something like `US$ 0` tax for free tickets

Reply:
And can you add a screenshot of situation if a discount code is applied

Reply:
@shreyanshdwivedi with the discount:

![image](https://user-images.githubusercontent.com/21087061/59055594-d6749980-88b3-11e9-85f6-7b7e38a11379.png)
and I will take care of the free tickets in next commit

Reply:
@uds5501 is your tax percent or amount? Of percent then it should be applied to the discounted amount. Right?

Reply:
@shreyanshdwivedi 
Tax is always percent rate. 
- The way i implemented it right now is calculate tax on base price of the ticket (which doesn't change)
- If a discount code is applied, it deducts the total price.

Reply:
@uds5501 if it is percent, we should ensure to apply tax on what user pays. This is basically followed everywhere.

Reply:
@shreyanshdwivedi okay, wip-ing it till i get this right.

Reply:
@uds5501 What will be the message in case tax is included in price and what about if we have added some ticket fees ?